go.mod

```diff
@@ -19,9 +19,9 @@ require (
 	gitlab.com/rodrigoodhin/gocure v0.0.0-20220718065339-f14dfe79276a
 	golang.org/x/exp v0.0.0-20230522175609-2e198f4a06a1
 	golang.org/x/time v0.3.0
-	google.golang.org/protobuf v1.31.0
-	istio.io/api v0.0.0-20230621030812-36698ea550c8
-	istio.io/client-go v1.18.1-0.20230621031510-0789e128a186
+	google.golang.org/protobuf v1.30.0
+	istio.io/api v0.0.0-20230713061407-06047cce866f
+	istio.io/client-go v1.18.1
 	istio.io/istio v0.0.0-20230621044911-6f0d7ae668c8
```

Comment:
IS there a reason why we didn't update `istio.io/istio`?

Reply:
😄 I remembered that we wanted to write a doc for updating Istio library when we do it the next time. 😏